tests/Aspire.Hosting.Tests/Python/AddPythonProjectTests.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.Tests.Helpers;
+using Xunit;
+using Microsoft.Extensions.DependencyInjection;
+using System.Text.RegularExpressions;
+
+namespace Aspire.Hosting.Tests.Python;
+
+public class AddPythonProjectTests
+{
+    private static readonly Regex s_pythonExecutablePattern  = new Regex("python(\\.(exe|bat|cmd|sh|ps1))?$", RegexOptions.IgnoreCase);
+    private static readonly Regex s_telemetryExecutablePattern = new Regex("opentelemetry-instrument(\\.(exe|bat|cmd|sh|ps1))?$", RegexOptions.IgnoreCase);
+    private static readonly string s_playgroundDirectory = Path.GetFullPath(Path.Combine(AppContext.BaseDirectory, "../../../../../playground/python/"));
+
+    [LocalOnlyFact("python")]
+    public void AddPythonProject_SetsResourcePropertiesCorrectly()
+    {
+        var pythonProjectDirectory = Path.Combine(s_playgroundDirectory, "script_only");
```

Comment:
You are going to need to use the testbuilder here (look at some of the other tests). Otherwise this will leak file watch handles.